cmd/cleanup/background.go

```diff
@@ -0,0 +1,41 @@
+// Copyright © 2024 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package cleanup
+
+import (
+	cx "context"
+	"time"
+
+	"github.com/ory/graceful"
+	"github.com/ory/kratos/driver"
+)
+
+func BackgroundCleanup(ctx cx.Context, r driver.Registry) {
+	ctx, cancel := cx.WithCancel(ctx)
+
+	r.Logger().Println("Cleanup worker started.")
```

Comment:
Suggestion [1]: Looks like this functionality is not used:
I didn't find `background-cleanup` configuration property to be set in `fandom-auth` repo 
There are no logs: https://kibana.wikia-inc.com/goto/194dda09e6eac2042e7980e78e6f1ccf
Cleanup job: https://github.com/Wikia/fandom-auth/blob/66cd955b826aabec89ac3d4e19d6ba54a340963e/cleanup.job.yaml#L74-L77

Reply:
those changes we're suggested at some point by ory and our changes were the same as a mainstream pr. I think we should go through our mainstream prs and check why those(and probably other) changes weren't merged. We could create a doc and follow-up with ory on some topics

Reply:
Follow-up task: https://fandom.atlassian.net/browse/PLATFORM-9042